lib/system/seqs_v2.nim

```diff
@@ -86,7 +95,7 @@ proc shrink*[T](x: var seq[T]; newLen: Natural) {.tags: [], raises: [].} =
     {.noSideEffect.}:
       cast[ptr NimSeqV2[T]](addr x).len = newLen
 
-proc grow*[T](x: var seq[T]; newLen: Natural; value: T) =
+proc grow*[T](x: var seq[T]; newLen: Natural; value: T) {.nodestroy.} =
```

Comment:
You cannot use `nodestroy` if not defined(nimHasDup).

Reply:
Then I guess it should be changed to

```nim
when defined(nimHasDup):
  proc grow*[T](x: var seq[T]; newLen: Natural; value: T) {.nodestroy.} =
else:
  proc grow*[T](x: var seq[T]; newLen: Natural; value: T) {.nodestroy.} =
```

Or we can simply update the csource_v2.

Reply:
Use a custom pragma to fight the code duplication.

Reply:
Make sense, though I doubt it fixes the CI failure.

Reply:
> You cannot use `nodestroy` if not `defined(nimHasDup)`

Why not ?